src/components/tabs/story/meta.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta } from '@storybook/react';
+import { Tabs } from 'src';
+
+export const createMeta = (title: string): Meta => {
+    return {
+        component: Tabs,
+        title,
+        argTypes: {
+            maxWidth: {
+                name: 'Content Width',
```

Comment:
No reason to have `Content Width` here.
Plz check how you can use Storybook controls for your component. You may add some stuff here and reduce stories you have.